plover/assets/main.json

```diff
@@ -34977,6 +34986,8 @@
 "KEUPB/SHA/SA": "Kinshasa",
 "KEUPB/SHAU/SA": "Kinshasa",
 "KEUPB/SHEUP": "kinship",
+"KEUPB/STET/EUBG": "kinesthetic",
```

Comment:
These two `ST-` strokes feel wrong to me. Are there other examples of this in the main dictionary?